.github/workflows/check-requirements.yml

```diff
@@ -0,0 +1,26 @@
+name: Check Requirements
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python 3.9
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install requests
+
+    - name: Check requirements
+      run: |
+        python tools/check-requirements.py requirements.txt
+        python tools/check-requirements.py requirements-dev.txt
```

Comment:
would this benefit from being combined with https://github.com/WIPACrepo/wipac-dev-tools/blob/main/.github/workflows/try-setup-install.yml?

Reply:
I actually like the idea of smaller independent workflows, for two reasons.

1. Failures are easy to classify.

If Try-Setup-Install fails, you know something failed during setup.  
If Check-Requirements fails, you know the requirements are lagging behind the latest version.

2. Multiple workflows can run in parallel.

Admittedly, GitHub has a cap (I think 4?) on the freebie runners, but parallel execution has the potential to run faster, so I'm happy to take advantage when it's available.

That said, there is the overhead of herding yaml workflows between projects.
A single "Do All The CI Thing" workflow can be more convenient for humans.

On the balance, I'd keep it separate, but I wouldn't oppose merging the two if you felt strongly about it.

Reply:
Separate is fine. Your CI step is more "local dev" vs "client use". They test different things, `try-setup-install` mimics clients with different py versions. There is _some_ overlap when `latest` == `setup.py`'s latest, but only for `requirements.txt`.